hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -29,6 +30,10 @@
   // TODO (elingg) better handling in livestate and persistent state of zkfc task. Right now they are
   // chained.
   private Task zkfcNodeTask;
+  //Timed Health Check for node health monitoring
+  private TimedHealthCheck healthCheckNN;
+  private TimedHealthCheck healthCheckZN;
```

Comment:
We don't need these private healthCheckNN members.  They can just be created when needed.  They aren't reference outside of the point at which they are launched.